book/website/reproducible-research/rdm/rdm-sharing.md

```diff
@@ -43,6 +43,7 @@ A few public data repositories are [Zenodo](https://zenodo.org/), [Figshare](htt
 So that others know what they can do with your data, you need to apply a licence [{term}`def<License>`] to your data.
 The most commonly used licences are [Creative Commons](https://creativecommons.org/choose/), [Open Government Licence](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/), or an [Open Data Commons Attribution License](https://opendatacommons.org/licenses/by/index.html).
 To get maximum value from data sharing, make sure that your paper and code both link to your data, and vice versa, to allow others understand your project better.
+See {ref}`rr-licensing` for more information. 
```

Comment:
I can't comment on line 39 above but you are welcome to list [Harvard Dataverse](https://dataverse.harvard.edu/) along with Zenodo, Figshare, and Dryad. We're all participating in the NIH's Generalist Repository Ecosystem Initiative (GREI) together: https://datascience.nih.gov/data-ecosystem/generalist-repository-ecosystem-initiative

Here is the full list from the NIH: https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/generalist-repositories (via https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/selecting-a-data-repository ).

Reply:
Added - thanks!